homeassistant/components/vera/__init__.py

```diff
@@ -227,6 +227,7 @@ def __init__(self, vera_device: DeviceType, controller_data: ControllerData):
     async def async_added_to_hass(self) -> None:
         """Subscribe to updates."""
         self.controller.register(self.vera_device, self._update_callback)
+        self.schedule_update_ha_state(True)
```

Comment:
Add the entities (`add_entities`) with second argument `True` instead. That will make sure that `update` is called during entity addition. There will always be a state update at the end of entity addition so we don't need to do an extra here.

Reply:
That's great - now changed.

Thanks.